### PR TITLE
[COMM-90] Add outline to focused elements

### PR DESCRIPTION
**Pull request by fredefox**

### CODE REVIEW

**Comment by fredefox** [comment]
Before: 
image

After: 
image

File: styles/_base.scss [hunk 1]
```diff
@@ -148,3 +144,8 @@ ul {
   width: 1px;
   white-space: nowrap;
 }
+
+:focus {
```
Comment (VilhelmNielsen) [inline styles/_base.scss]: I don't think this will work.

Comment (fredefox) [reply styles/_base.scss]: I think you're right. I dun goofed.

**Comment by augustocravosilva** [comment]
Closing this one for now. We might need to revisit focus styles fixes in  the v2 version.

**Pull request closed without merging**